tests/prague/eip7692_eof_v1/eip3540_eof_v1/test_all_opcodes_in_container.py

```diff
@@ -88,6 +89,26 @@ def make_opcode_valid_bytes(opcode: Opcode) -> Opcode | Bytecode:
     return code
 
 
+def create_dependant_container(opcode: Opcode) -> List[Section]:
```

Comment:
I just reworked this file in https://github.com/ethereum/execution-spec-tests/pull/650 so there might be conflicts, although the approach in that PR was to simply skip `EOFCREATE` and `RETURNCONTRACT` (my thinking was that this is tested elsewhere anyway).